SayTheirNames/Source/Dependency/DependencyContainer.swift

```diff
@@ -24,23 +24,63 @@
 
 import UIKit
 
-protocol Servicing {
-    var image: ImageService { get }
-    var dateFormatter: DateFormatterService { get }
-    var network: NetworkRequestor { get }
+// MARK: - Dependency
+protocol Dependency {
+    
 }
-/// This is a core class that holds all instances responsible for logic
-final class Service: Servicing {
+
+// MARK: - DependencyContainer
+final class DependencyContainer: Dependency {
+    lazy private(set) var navigator = Navigator()
```

Comment:
can it be `let` ? the deps should be lightweight to create so I think it's ok if not lazy

Reply:
We can indeed. Only thing is we cannot have a `Dependency` use `@DependencyInject`.

Reply:
I think as the amount of dependencies grows, using lazy would be alright so we aren't bringing these services into memory. Overall, I can go either way

Reply:
If we go lazy we can have a `Dependency`use `@DependencyInject`. Im ok with either our.